File: viewer/src/main/java/com/tianxing/hotflyer/viewer/fragment/ReleasedFragment.java
package com.tianxing.hotflyer.viewer.fragment;

import com.tianxing.hotflyer.viewer.JAViewer;

import okhttp3.ResponseBody;
import retrofit2.Call;

/**
 * Project: JAViewer
 */
public class ReleasedFragment extends MovieFragment {
    @Override
    public Call<ResponseBody> newCall(int page) {
        return JAViewer.SERVICE.getReleased(page);
    }
}

File: viewer/src/main/java/com/tianxing/hotflyer/viewer/fragment/ReleasedFragment.kt
package com.tianxing.hotflyer.viewer.fragment

import com.tianxing.hotflyer.viewer.JAViewer

import okhttp3.ResponseBody
import retrofit2.Call

/**
 * Project: JAViewer
 */
class ReleasedFragment : MovieFragment() {
    override fun newCall(page: Int): Call<ResponseBody> {
        return JAViewer.SERVICE.getReleased(page)
    }
}
